Branch-SDK/Branch-SDK/Requests/BranchOpenRequest.m

```diff
@@ -173,4 +176,52 @@ - (NSString *)getActionName {
     return @"open";
 }
 
+
+#pragma - Open Response Lock Handling
+
+
+//	Instead of semaphores, the lock is handled by scheduled dispatch_queues.
```

Comment:
By habit I use a tab.  I frequently see people use two spaces after the comment delimiter too.

On a broader note, I started an experimental SDK project that has a clang-format definition file.  The format file is called '.clang-format'.

The project is here:  https://github.com/BranchMetrics/BranchLabs-iOS

We can play with formatting there before deciding.  But I think having a formatter is a good idea.